raiden/blockchain/state.py

```diff
@@ -11,6 +11,7 @@ def get_channel_state(
         reveal_timeout,
         payment_channel_proxy,
         opened_block_number,
+        chain_id,
```

Comment:
cant we extend `detail` to return  the `chainid`? IMO it makes more sense, since we really want to be using the value that is defined in the smart contract.